static/src/js/App.jsx

```diff
@@ -87,7 +88,7 @@ export const App = () => {
       const { items } = acls
 
       if (items.length > 0) {
-        const providerList = items.map(({ acl }) => acl.provider_identity.provider_id)
+        const providerList = items.map((item) => item.providerIdentity.provider_id)
```

Comment:
@dmistry1 is this still going to be snake cased? Your GQL PR calls camelcaseKeys in a few spots.

Reply:
Yes this provider is still going to be snakedCase, the camelcaseKeys I has was only for groups and catalogIdentity

Printed out the response from graphql to verify.
![image](https://github.com/nasa/mmt/assets/67551614/7e515bed-c7e8-489b-b192-268ed81dcfe4)